recipes/wg21-linear_algebra/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
-  "0.7.2":
-    url: https://github.com/BobSteagall/wg21/archive/refs/tags/v0.7.2.tar.gz
-    sha256: 4a192d279491264be2d687f89c15432fcb4f2e2d113599152bd3df2df42ce9e1
+  "0.7.3":
```

Comment:
Was there a particular reason to drop the old version? we generally try to leave them -- especially since this would update it to v2 which would be great

Reply:
I saw it worked https://github.com/conan-io/conan-center-index/pull/16392#issuecomment-1493068141

Reply:
I intended to release 0.7.2 but as this package depends on MDSpan, MDSpan removed support for version 0.1.0 (in this PR upgrading MDSpan to Conan 2.0: https://github.com/conan-io/conan-center-index/pull/15464/files) which was required by version 0.7.2.  

However to support MDSpan 0.5.0 I've had to update this to the latest version of changes in the Linear Algebra repository (and so pushed out release 0.7.3 to get access to these changes).  Without version 0.1.0 of MDspan, the old 0.7.2 release will not compile.